src/libraries/System.Net.Http/src/System/Net/Http/HttpTelemetry.cs

```diff
@@ -0,0 +1,104 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics.Tracing;
+using System.Runtime.CompilerServices;
+using System.Threading;
+
+// TODO: Get rid of this once I figure out how to handle build errors like these given the current lazy creation pattern:
+//     System\Net\Http\HttpTelemetry.cs(19,32): error CS8618: Non-nullable field '_startedRequestsCounter' is uninitialized.
+//     Consider declaring the field as nullable. [S:\GitHub\runtime\src\libraries\System.Net.Http\src\System.Net.Http.csproj]
+#nullable disable
```

Comment:
You can do `_startedRequestsCounter = null!` if you know they'll be initialized later prior to use.

Reply:
I tried that. But then I got compile errors from the nullable checks. Basically I didn't want to declare these fields as nullable. The logic is such that they are not null when not being used.

@stephentoub Is there a way to handle this in the logic without making the fields nullable?

Reply:
_startedRequestsCounter should be nullable:
```C#
private PollingCounter? _startedRequestsCounter;
```
What problems does that cause?